<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logindia</class>
 <widget class="QDialog" name="Logindia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
    background-image: url(:/new/images/login.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:transparent;
	background-image: url(:/new/images/dl2.png);
}
QPushButton:hover
{
	background-image: url(:/new/images/dl1.png);
}
QPushButton:pressed
{
	background-image: url(:/new/images/dl3.png);
}
/*QPushButton#pushButton
{
	background:url(E:/dl1.png);    
}

QPushButton#pushButton:hover
{
	background:url(E:/loginbutton2.png);    
}*/





</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:transparent;
	background-image: url(:/new/images/cc2.png);
}
QPushButton:hover
{
	background-image: url(:/new/images/cc1.png);
}
QPushButton:pressed
{
	background-image: url(:/new/images/cc1.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
    color:#9BC200;
}</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
    color:#9BC200;
}</string>
   </property>
   <property name="text">
    <string>密 码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
